modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -93,14 +93,17 @@ test.describe('Manage object fields through Model Builder', () => {
 		modelBuilderPage,
 	}) => {
 		const {listTypeDefinitionIds, objectDefinition} = createdEntities;
-
-		const listTypeDefinitions = await Promise.all(
-			Array(22)
-				.fill(null)
-				.map(() =>
-					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
-				)
-		);
+				
+		let listTypeDefinitions = (await apiHelpers.listTypeAdmin.getListTypeDefinitions()).items;
+
+		listTypeDefinitions = listTypeDefinitions.concat(
```

Comment:
Hey @p-albuquerque, I think it's better to create a new const with a different name, something like `allListTypeDefinitions`